### DISCUSSION TITLE
image upload error 413

### DISCUSSION THREAD

**Discussion by ElliotHYLee** [pocketbase/pocketbase#2337] [Q&A] [answered]
I have update the old project with PB 0.15.0. Smooth.

I created a new PB project with 0.15.0. It has an odd bahavior with uploading an image of 1.4mb with the PB's admin panel. At this point, I'm not even sure if it's due to the exact sizes or anything else. Other small sized images (~ a few hundred KB) have no issues.

my image field setup (the same setup as old project, which doesn't have any issue)
allowed mime types: no restriction
max file size: 10242880 (~10mb)
max select: 10000000

when i upload an image it says: 
Something went wrong while processing your request.

**From the F12 console (Chrome):**
(anonymous) @ index-7275477a.js:1
index-7275477a.js:4     PATCH [MY_ENDPOINT]/pb/api/collections/t6clmy3h7jtn1qy/records/p387s6jnznoyuyd 413
(anonymous) @ index-7275477a.js:4
(anonymous) @ index-7275477a.js:4
(anonymous) @ index-7275477a.js:4
(anonymous) @ index-7275477a.js:4

The old project with the PB 0.15 doesn't have this issue. So, I exported and imported the schema from the old data. In the new project, it still shows the problme.

My guess is that 0.15.0 does not correspond with the image size setting? But if so, the old DB with v0.15.0 should have the same issue. I really have no idea how to even explain and/or reproduce. Any advise to probe what's the potential cause?

**Comment by ganigeorgiev** [comment, marked as answer]
We don't explicitly throw 413 errors.

Do you have a reverse proxy, eg. nginx? If yes, make sure that the `client_max_body_size` or similar directive is properly configured.
